Tests/UserServiceTests/Commands/CreateUserCommandTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using NUnit.Framework;
+using UserService.Interfaces;
+using UserService.Commands;
+using FluentValidation;
+using DTO.RestRequests;
+using Moq;
+using MassTransit;
+using DTO.BrokerRequests;
+using DTO;
+using System.Threading.Tasks;
+using FluentValidation.Results;
+using Kernel.CustomExceptions;
+
+namespace UserServiceTests
+{
+   public class CreateUserCommandTests
+    {
+        private ICreateUserCommand command;
+        private Mock<IValidator<CreateUserRequest>> validatorMock;
+        private Mock<IRequestClient<InternalCreateUserRequest>> clientMock;
+        private Mock<Response<OperationResult>> responseMock;
+        private Mock<ValidationResult> validatorResultMock;
+        private CreateUserRequest request;
+        
+        [SetUp]
+        public void Initialization()
+        {
+            request = new CreateUserRequest();
+            request.Password = "Not empty string";
+
+            validatorResultMock = new Mock<ValidationResult>();
+            validatorResultMock
+                .Setup(x => x.IsValid)
+                .Returns(true);
+
+            validatorMock = new Mock<IValidator<CreateUserRequest>>();
+            validatorMock
+                .Setup(x => x.Validate(It.IsAny<ValidationContext>()))
+                .Returns(validatorResultMock.Object);
+                
+            responseMock = new Mock<Response<OperationResult>>();
+            responseMock
+                .Setup(x => x.Message)
+                .Returns(new OperationResult() { IsSuccess = true});
+
+            clientMock = new Mock<IRequestClient<InternalCreateUserRequest>>();
+            clientMock
+                .Setup(x => x.GetResponse<OperationResult>(It.IsAny<InternalCreateUserRequest>(), default, default))
+                .Returns(Task.FromResult(responseMock.Object));
+
+            command = new CreateUserCommand(clientMock.Object, validatorMock.Object);
+        }
+
+        [Test]
+        public void CreateUserCommand_ValidationException_IncorrectUserData()
```

Comment:
Names of methods should be in CamelCase

Reply:
> Names of methods should be in CamelCase

https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices

So even at Microsoft they call tests

Reply:
> > Names of methods should be in CamelCase
> 
> https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> 
> So even at Microsoft they call tests

Where do they say about underlines?

Reply:
> > > Names of methods should be in CamelCase
> > 
> > 
> > https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> > So even at Microsoft they call tests
> 
> Where do they say about underlines?


They call all tests so

Reply:
> > > > Names of methods should be in CamelCase
> > > 
> > > 
> > > https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> > > So even at Microsoft they call tests
> > 
> > 
> > Where do they say about underlines?
> 
> They call all tests so

And where in the recommendations they say "call tests using underlines?"

Reply:
> And where in the recommendations they say "call tests using underlines?"


They don’t write like that. But they do it

Reply:
> Names of methods should be in CamelCase

Fixed

Reply:
> > > > > Names of methods should be in CamelCase
> > > > 
> > > > 
> > > > https://docs.microsoft.com/ru-ru/dotnet/core/testing/unit-testing-best-practices
> > > > So even at Microsoft they call tests
> > > 
> > > 
> > > Where do they say about underlines?
> > 
> > 
> > They call all tests so
> 
> And where in the recommendations they say "call tests using underlines?"

Fixed

Reply:
But only you use underscores in names of test methods. I think, it`s not good practise

Reply:
> But only you use underscores in names of test methods. I think, it`s not good practise


I have corrected

Reply:
Sorry, I didn`t see it